tv_apps/tv_shared/components/smart-banner/style.css

```diff
@@ -102,7 +113,24 @@ smart-banner.opening:before {
 }
 
 smart-banner.closing {
+  transition-property: all;
   transition-duration: 0.5s;
+  padding-left: 11rem;
   padding-right: 0;
   background-color: rgba(0, 0, 0, 0.5);
 }
+
+smart-banner.hiding {
+  opacity: 0;
+  transition-property: opacity;
+  transition-duration: 1s;
+}
+
+smart-banner.flying {
+  animation: flyAnimation 1s;
+}
+
+@keyframes flyAnimation {
```

Comment:
This kind of design gives us "shaking" behavior. This is unexpected. Does Scott accept this?